.github/workflows/ci.yml

```diff
@@ -814,8 +814,12 @@ jobs:
       run: .github/scripts/get-latest-cs.sh
       shell: bash
     - name: Generate native launcher
-      run: .github/scripts/generate-native-image.sh
-      shell: bash
+      uses: nick-fields/retry@v3
+      with:
+        timeout_minutes: 30
+        max_attempts: 5
+        shell: bash
+        command: .github/scripts/generate-native-image.sh
```

Comment:
Shouldn't we instead do it in the bash script?

Reply:
...does it make any real difference?

Reply:
maybe not, I might be too careful here